platform-includes/sourcemaps/legacy-troubleshooting/javascript.mdx

```diff
@@ -70,7 +70,7 @@ If your `sourceMappingURL` comment is similar to:
 An example `sentry-cli` command to upload these files correctly would look like this (assuming you're in the `/scripts` directory, running your web server from one directory higher, which is why we're using the `--url-prefix` option):
 
 ```shell
-sentry-cli releases files VERSION upload-sourcemaps . --url-prefix '~/scripts'
+sentry-cli sourcemaps upload --release VERSION --url-prefix '~/scripts' .
```

Comment:
```suggestion
sentry-cli sourcemaps upload --url-prefix '~/scripts' .
```

Is the release required here for any specific reason, or was it only present before because it was required with the old API? If we don't need it, let's get rid of it.

This same comment applies to the other changes in this file

Reply:
@loewenheim maybe do you know the background?

Reply:
@loewenheim, so we can likely remove the release now if I understand correctly?

Reply:
Well, it depends on if you're uploading with or without debug ids. Users should ideally use debug ids, but AFAIK it's not mandatory.

Reply:
Okay, I will remove `--release` then. I think the CLI will yell at them anyways if they are missing a release when they need it

Reply:
Sounds good, yeah.